packages/components/src/__actions__/Button/v3/Button.tsx

```diff
@@ -9,9 +9,7 @@ import { ButtonContent, PendingContent } from "./subcomponents"
 import { ButtonSizes, ButtonVariants, PendingButtonProps } from "./types"
 import styles from "./Button.module.css"
 
-type ButtonBaseProps = Omit<RACButtonProps, "children"> & {
-  /** Used as the label for the button. */
-  children: RACButtonProps["children"]
+export type ButtonBaseProps = {
```

Comment:
Mmmm I get where you're coming from but its not necessarily style focus since we've got stuff like icon in there 🤔 I'm think we could use something like`ButtonSharedProps` `ButtonCommonProps` we could assume that most buttons would have these? Maybe even `ButtonUIProps`?

Reply:
`ButtonUIProps` sounds good :) I consider icon stylistic in this case thus ButtonStyle, but UI is good too